scripts/run_local.sh

```diff
@@ -29,16 +29,16 @@ VALIDATOR_FILES=()
 for i in $(seq 1 $NUM_VALIDATORS); do
     VALIDATOR_FILES+=("$CONF_DIR/validator_$i.toml")
 done
-./linera-server generate --validators "${VALIDATOR_FILES[@]}" --committee committee.json
+./linera-server generate --validators "${VALIDATOR_FILES[@]}" --committee committee.json --testing-prng-seed 1
```

Comment:
In this case you probably also want it in `LocalNetwork::generate_initial_validator_config` which is used in `linera net up`.

Also, two ideas, less flexible but perhaps cleaner:
1. Confine `testing-prng-seed` to only debug versions of the binary so that it does not get misused accidentally in production
2. If the aim is reproducibility, why be able to attach an arbitrary seed and not always seed to 0?

Reply:
@christos-h Yes, now, I have updated linera net up

Reply:
re: debug version: I'm open to the idea of using the feature `test` for this.

Reply:
What I like about feature-gating is that we could use an environment variable instead of a CLI flag in good conscious.